APPLICATION.md

```diff
@@ -0,0 +1,42 @@
+# Application Notes
+
+This Applicatio Notes is targeted at both model packagers and application developers and is for infomation that is not directly relevant to users who are just simply trying to use llamafiles in a standalone manner. Instead it is for developers who want their models to better integrate with other developers models or applications. (Think of this as an informal ad-hoc community standards page)
+
+## Finding Llamafiles
+
+While we do not have a package manager for llamafiles, applications developers are recommended
+to search for AI models tagged as `llamafile` in Hugging Face AI repository.
+Be sure to display the publishing user or organisation and to sort by heart count.
```

Comment:
I would revise my early suggestion to "sort by trending" so we don't end up with permanent victors, and people have a better chance of being able to talk online with their friends about what's popular at any given moment.